docs/commerce/checkout/configure_checkout.md

```diff
@@ -0,0 +1,237 @@
+                    money_transfer:
+                        name: "Money transfer"
+```
+
+## Shipping and billing address field format configuration 
+
+In your implementation, you may need to create custom format configurations 
+for the shipping or billing address fields, for example, to use different address 
+formats based on the buyer's geographical location.
+
+Field formats for the billing and shipping addresses comply with the [FieldType Address](../../content_management/field_types/field_type_reference/addressfield.md#formats) specification and can be controlled with the `billing_address_format` and `shipping_address_format` flags, respectively.
+They fallback to `billing` and `shipping` predefined formats by default:
+
+- `billing` is sourced from the `corporate-accounts` bundle 
+- `shipping` is part of the `checkout` bundle's default configuration 
+
+To modify address formats you can either override the existing formats or create 
+custom ones.
+
+### Override existing Address Field Type formats 
+
+To override the existing `billing` and `shipping` address Field Type formats, edit 
+the `../vendor/ibexa/checkout/src/bundle/Resources/config/fieldtype_address.yaml` 
+file, and modify the address format, for example, by adding a tax identifier 
+to the `billing` address:
+
+``` yaml 
+formats:
+    shipping:
+        country:
+            default:
+                - region
+                - locality
+                - street
+                - postal_code
+                - email
+                - phone_number
+                
+    billing:
+        country:
+            default:
+                - region
+                - locality
+                - street
+                - postal_code
+                - email
+                - phone_number
+                - tax_number
+```
+
+!!! note
+
+    If you are overriding the default formats and not changing their default names, 
+    there is no need to modify the keys under `ibexa.repositories.<repository_name>.checkout`.
+
+### Define custom Address Field Type formats 
+
+To create custom Address Field Type formats to be used in Checkout, make the following changes in the configuration files. 
+
+First, under under `ibexa.repositories.<repository_name>.checkout`, define custom 
+format configuration keys for `billing_address_format` and `shipping_address_format`:
+
+``` yaml 
+ibexa:
+    repositories:
+        <repository_name>:
+            checkout:
+                billing_address_format: <custom_billing_fieldtype_address_format_identifier> #coming from Corporate Account "billing" by default
+                shipping_address_format: <custom_shipping_fieldtype_address_format_identifier> #coming from Corporate Account "shipping" by default 
+                customer_content_type: <your_ct_identifier_for_customer> #customer by default, it is used in registration and uses given shipping/billing addresses to pre-populate address forms in "Select Address" checkout step
```

Comment:
Best move the comments to lines above, otherwise the code block won't be readable